packages/rum-core/src/domain/resource/matchRequestTiming.ts

```diff
@@ -2,13 +2,17 @@ import type { Duration, RelativeTime } from '@datadog/browser-core'
 import { addDuration } from '@datadog/browser-core'
 import type { RumPerformanceResourceTiming } from '../../browser/performanceCollection'
 import type { RequestCompleteEvent } from '../requestCollection'
-import { toValidEntry } from './resourceUtils'
+import { isValidEntry } from './resourceUtils'
 
 interface Timing {
   startTime: RelativeTime
   duration: Duration
 }
 
+// we use a WeakMap because WeakSet is not supported in ie11
+const PLACEHOLDER = 1
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could extract a weakset polyfill doing the weakmap + placeholder trick to remove this concern from this file